src/main/java/com/online/bookstore/model/Role.java

```diff
@@ -0,0 +1,24 @@
+package com.online.bookstore.model;
+
+import com.online.bookstore.model.enumeration.RoleName;
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
do not use Data with Entities and add soft delete

Reply:
fixed